src/QuantumSdk/DefaultItems/DefaultItems.props.v.template

```diff
@@ -34,6 +34,9 @@
     <IncludeQsharpCorePackages>true</IncludeQsharpCorePackages> 
     <IncludeProviderPackages>true</IncludeProviderPackages> 
     <QsharpDocsGeneration>false</QsharpDocsGeneration>
+    <QsharpDocsPackageId>$(PackageId)</QsharpDocsPackageId>
+    <!-- Can be "yaml" for DocParser-based, or "markdown" for RewriteStep-based. -->
+    <QsharpDocGenerationMode>markdown</QsharpDocGenerationMode>
```

Comment:
Do we really want to permanently maintain both? I'd rather go with one docs generation tool if possible rather than maintain both. 
If we were to keep this, then it needs to be documented in the readme of the Sdk, and ideally we would fail the build or at least emit a warning if this is not set to a recognized mode.